lib/exercises.rb

```diff
@@ -1,29 +1,60 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(mn)
+# Space Complexity: O(mn)
 
 def grouped_anagrams(strings)
```

Comment:
Technically since you're sorting the strings its O(n * m log m) where m is the length of the strings, but if you assume the strings are going to be short this is O(n)